packages/cli/src/build/config.ts

```diff
@@ -198,14 +220,19 @@ export function getRollupConfigs(pkg: Package, aliases: Aliases) {
           interop,
           plugins: cjsPlugins,
         },
-        hasModuleField && {
-          format: "es" as const,
-          entryFileNames: `[name].${getDistExtension("browser-esm")}`,
-          chunkFileNames: `dist/[name]-[hash].${getDistExtension(
-            "browser-esm"
-          )}`,
-          dir: pkg.directory,
-        },
+        ...(hasModuleField
```

Comment:
I'm not sure why `browser` configs depend on `hasModuleField` here but perhaps it's always validated to be true when `exports` are enabled?